golem-api-grpc/proto/golem/worker/worker_service.proto

```diff
@@ -47,7 +46,7 @@ message LaunchNewWorkerRequest {
 
 message LaunchNewWorkerResponse {
   oneof result {
-    golem.worker.VersionedWorkerId success = 1;
+    golem.worker.WorkerId success = 1;
```

Comment:
actually i think it is quiet useful, if this returning version of created worker, maybe instead there should be 

```proto
message LaunchNewWorkerSuccess {
    golem.worker.WorkerId worker_id = 1;
    uint64 template_version = 2;
    ...
}

message LaunchNewWorkerResponse {
  oneof result {
    LaunchNewWorkerSuccess success = 1;
      ...
   }
}
```
also in such way it is more prepared for future extensions of response

Reply:
Good idea, thanks